api/rds.proto

```diff
@@ -262,6 +262,11 @@ message RedirectAction {
   string path_redirect = 2;
 }
 
+message TracingDecorator {
```

Comment:
Thoughts on just naming this "Decorator" ? As I said before I think this may end up getting used beyond tracing. For example we could later add an option to add dynamic stats for this decorated route. Useful for logging also potentially.